iroh-cli/Cargo.toml

```diff
@@ -40,12 +40,12 @@ human-time = "0.1.6"
 indicatif = { version = "0.17", features = ["tokio"] }
 iroh = { version = "0.15.0", path = "../iroh", features = ["metrics"] }
 iroh-metrics = { version = "0.15.0", path = "../iroh-metrics" }
+iroh-net = { version = "0.15.0", path = "../iroh-net" }
```

Comment:
no need for this, you can just do `iroh::net`

Reply:
good point, done in 0c303f1